generator/src/lib.rs

```diff
@@ -2805,20 +2817,25 @@ pub fn constant_name(name: &str) -> &str {
 }
 
 pub fn generate_constant<'a>(
-    constant: &'a vkxml::Constant,
-    cache: &mut HashSet<&'a str>,
+    constant: &'a vk_parse::Enum,
+    global_const_cache: &mut HashMap<&'a str, CType>,
 ) -> TokenStream {
-    cache.insert(constant.name.as_str());
-    let c = Constant::from_constant(constant);
+    let (c, _, is_alias) = Constant::from_vk_parse_enum(constant, None, None).unwrap();
+    // This requires knowing the type of the alias, that is unavailable here.  Besides
+    // backwards-compatibility, stabilization aliases serve no purpose.
     let name = constant_name(&constant.name);
     let ident = format_ident!("{}", name);
     let notation = constant.doc_attribute();
-
-    let ty = if name == "TRUE" || name == "FALSE" {
+    let ty = if let Constant::Alias(a) = &c {
+        // TODO: Drop is_alias over Constant::Alias match
+        assert!(is_alias);
+        global_const_cache[a.to_string().as_str()]
```

Comment:
It looks like this code both populates and reads `global_const_cache`. How do we know the value we're trying to read has been populated already? Document.